lib/dom.js

```diff
@@ -751,6 +752,20 @@ declare class HTMLFormElement extends HTMLElement {
     submit(): void;
 }
 
+declare class HTMLIFrameElement extends HTMLElement {
+    allowFullScreen: boolean;
+    frameBorder: number;
+    height: number;
+    marginHeight: number;
+    marginWidth: number;
+    name: string;
+    scrolling: string;
+    sandbox: string;
```

Comment:
https://html.spec.whatwg.org/multipage/embedded-content.html#the-iframe-element says this is a DOMTokenList but I don't know what that is